### ISSUE TITLE
Outdated links to UNDERSTANDING-WCAG20 in WCAG 2.1 and 2.2

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1828]
# Outdated links to UNDERSTANDING-WCAG20 in [WCAG 2.1](https://www.w3.org/TR/WCAG21/)

* [5. Conformance](https://www.w3.org/TR/WCAG/#conformance): link to [Understanding Conformance](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance)
* [5.2.4 Only Accessibility-Supported Ways of Using Technologies](https://www.w3.org/TR/WCAG/#cc4): link to [Understanding accessibility support](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance)
* [5.2.5 Non-Interference](https://www.w3.org/TR/WCAG/#cc5): link to [Understanding Conformance Requirements](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance)
* [5.3.2 Optional Components of a Conformance Claim](https://www.w3.org/TR/WCAG/#conformance-optional): in first note, link to [Understanding Conformance Claims](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance), second note link to [Understanding metadata](https://www.w3.org/TR/UNDERSTANDING-WCAG20/appendixC)
* [accessibility supported](https://www.w3.org/TR/WCAG/#dfn-accessibility-supported) glossary definition: in first note, link to [Level of Assistive Technology Support Needed for "Accessibility Support"](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance); fifth note, link to [Understanding Accessibility-Supported Web Technology Uses](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance)
* [relative luminance](https://www.w3.org/TR/WCAG/#dfn-relative-luminance) glossary definition: in second note, link to [Understanding Success Criterion 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast)
* [text alternative](https://www.w3.org/TR/WCAG/#dfn-text-alternative) glossary definition: in note, link to [Understanding Text Alternatives](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance)

These should be amended as errata.

# Outdated links to UNDERSTANDING-WCAG20 in [WCAG 2.2](https://www.w3.org/TR/WCAG22/)

* [relative luminance](https://www.w3.org/TR/WCAG/#dfn-relative-luminance) glossary definition: in second note, link to [Understanding Success Criterion 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast)
* [text alternative](https://www.w3.org/TR/WCAG/#dfn-text-alternative) glossary definition: in note, link to [Understanding Text Alternatives](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance)

These should be amended before 2.2 goes to publication

**Comment by alastc** [comment]
> These should be amended as errata.

At least some of the ones in 2.1 were updated ages ago (probably everything except the two under 2.2), but the "Official" document is form June 2018 and doesn't tend to get updated.

I don't _think_ that updated links need to count as an errata (otherwise there would be plenty on the errata page), but whether they will appear on the /wcag21/ version I'm not sure.

**Comment by patrickhlauke** [comment]
> the "Official" document is from June 2018 and doesn't tend to get updated

so does it remain preserved in amber now, or can a small non-normative update not be pushed as erratum?

**Comment by alastc** [comment]
So I think all the instances in the WCAG 2.1 code were gone ages ago. The only other instances of "UNDERSTANDING-WCAG20" were in the index page (supposed to go to the old version), and another one I found in the aria.html page.

@michael-n-cooper does the 'local' link I've used in the first file work in definition files?
https://github.com/w3c/wcag/pull/1830/files 

> so does it remain preserved in amber now, or can a small non-normative update not be pushed as erratum?

Erratum can be published, but then you have to refer to the erratum page, it doesn't update the spec on the face of it. (Not an approach I agree with BTW.) Apparently changing the date on the face of the spec causes panic in certain quarters.

FWIW, I think link updates are editorial if they go to the later equivalent resource, so don't need an errata.

**Comment by patrickhlauke** [comment]
yeah, agree it's not really an errata, was only a suggestion in case THAT triggered a republication. but if it doesn't...oh well
